tests/cursor.rs

```diff
@@ -0,0 +1,115 @@
+mod utils;
+
+use ion_binary_rs::IonValue;
+use qldb::DocumentCollection;
+use qldb::QLDBClient;
+use rand::distributions::Alphanumeric;
+use rand::thread_rng;
+use rand::Rng;
+use utils::ensure_test_table;
+
+#[async_std::test]
+async fn cursor_800_documents() {
+    let client = QLDBClient::default("rust-crate-test").await.unwrap();
+
+    let test_table = ensure_test_table(&client).await;
+
+    let documents_model = format!("cursor_800_documents{}", rand_string());
+
+    for _ in 0..20 {
```

Comment:
I think is better to use a well named variable than a magical number like 20.

Reply:
Yep! Updating!